ui/src/components/nodes/Code.tsx

```diff
@@ -278,19 +275,69 @@ export const ResultBlock = memo<any>(function ResultBlock({ id, layout }) {
               <Ansi>{stdout}</Ansi>
             </Box>
           )}
-          {result?.text && result?.count > 0 && (
+          {results && results.length > 0 && (
             <Box
               sx={{
                 display: "flex",
                 fontSize: "0.8em",
-                flexDirection: "row",
-                alignItems: "center",
+                flexDirection: "column",
+                alignItems: "left",
                 borderTop: "1px solid rgb(214, 222, 230)",
               }}
             >
-              <Box component="pre" whiteSpace="pre-wrap">
-                {result.text}
-              </Box>
+              {results.map((res, i) => {
+                switch (res.type) {
+                  case "stream_stdout":
+                    return (
+                      <Box
+                        component="pre"
+                        whiteSpace="pre-wrap"
+                        key={i + 1}
```

Comment:
The key must be unique. There's a console warning.

<img width="767" alt="Screenshot 2023-08-05 at 1 07 50 PM" src="https://github.com/codepod-io/codepod/assets/4576201/9cb7be5d-d8ca-41c0-960b-2415aa5a0420">